autoload/EasyMotion/migemo/cp932.vim

```diff
@@ -1,56 +1,97 @@
 function! EasyMotion#migemo#cp932#load_dict()
     return {
```

Comment:
I use a following shell script to generate it.
It generates migemo-regexps for every ASCII chars except whitespace, and then escape `'`.

https://gist.github.com/kmorihiro/27035356e5e194a9e05411021e809ad3

This script requires migemo-dict files to generate regexps.
Each migemo-dict (cp932, eucjp, utf8) is genereted from current cmigemo master branch.

https://github.com/koron/cmigemo/tree/5c014a885972c77e67d0d17d367d05017c5873f7